### ISSUE TITLE
Outdated docs for Phalcon Devtools

### ISSUE THREAD

**Issue by rogervila** [phalcon/docs#869]
Hello.

The [Phalcon site tutorial](https://docs.phalconphp.com/en/latest/reference/tutorial.html) is not updated with examples from version 3.

The same goes for the [tutorial about devtools](https://docs.phalconphp.com/en/latest/reference/tools.html)

This is [the original issue](https://github.com/phalcon/phalcon-devtools/issues/774) that I posted at [Phalcon Devtools repository](https://github.com/phalcon/phalcon-devtools).

Thank you.

**Comment by SidRoberts** [comment]
I've updated the versions numbers and command outputs in the Dev Tools reference (3bf7dac3773f8bb4b51c30360eebaa839f4c449c, 58c9f1b43949ca1e343aa0977a48b087d0dbbd32).

I've also made some changes to the Tutorial reference but can't think of anything that specifically need's updating. What's out of date in the Tutorial reference?

As the official site takes a while to update, here are the latest versions:

- http://phalcon-php-framework-documentation.readthedocs.io/en/latest/reference/tutorial.html
- http://phalcon-php-framework-documentation.readthedocs.io/en/latest/reference/tools.html
